package.json

```diff
@@ -17,8 +17,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "compile-css": "node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 scss/style.scss css/styles.css",
+    "generate-icons-map": "node ./src/icons/generateIconsMap",
```

Comment:
More file naming: `generateIconsMap` -> `generate-icon-map` (No camelCase in file names.)